src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -898,12 +898,14 @@ private void CheckFeatureAvailability(MessageID feature)
                 return;
             }
 
-            if (feature.RequiredFeature() != null)
+            var requiredFeature = feature.RequiredFeature();
+            if (requiredFeature != null)
             {
                 if (!options.IsFeatureEnabled(feature))
                 {
-                    this.AddError(ErrorCode.ERR_FeatureIsExperimental, feature.Localize());
+                    this.AddError(ErrorCode.ERR_FeatureIsExperimental, feature.Localize(), requiredFeature);
```

Comment:
I'll fix that, but I can't test it, as none of its call sites are for features that have feature flags.